line-bot-api-client/src/main/java/com/linecorp/bot/client/LineMessagingService.java

```diff
@@ -16,19 +16,13 @@
 
 package com.linecorp.bot.client;
 
+import java.util.concurrent.CompletableFuture;
+
 import com.linecorp.bot.model.PushMessage;
 import com.linecorp.bot.model.ReplyMessage;
 import com.linecorp.bot.model.profile.UserProfileResponse;
 import com.linecorp.bot.model.response.BotApiResponse;
 
-import okhttp3.ResponseBody;
-import retrofit2.Call;
-import retrofit2.http.Body;
-import retrofit2.http.GET;
-import retrofit2.http.POST;
-import retrofit2.http.Path;
-import retrofit2.http.Streaming;
-
 public interface LineMessagingService {
```

Comment:
If you want to break the backward compatibility, you need to use NEW class name instead.